packages/reports/src/Containers/reports.tsx

```diff
@@ -88,14 +109,16 @@ const Reports = ({
                                 className='reports__route-selection'
                                 list_items={menu_options().map(option => ({
                                     text: option.label,
-                                    value: option.path,
+                                    value: option.path ?? '',
                                 }))}
-                                value={selected_route.path}
+                                value={selected_route.path ?? ''}
                                 should_show_empty_option={false}
                                 onChange={handleRouteChange}
+                                label={''}
```

Comment:
These props (`label` and `hide_top_placeholder`) are required in `SelectNative` component.